k8_cortx_cloud/deploy-cortx-cloud.sh

```diff
@@ -629,7 +632,12 @@ function deployCortxConfigMap()
     # Generate config files
     for i in "${!node_name_list[@]}"; do
         new_gen_file="$auto_gen_path/config.yaml"
-        cp "$cfgmap_path/templates/config-template.yaml" $new_gen_file
+        if [ "$SERVER_POD_DEPLOYMENT" == true ]; 
```

Comment:
This should be DATA_POD_DEPLOYMENT, i.e. 
```
if [ "$SERVER_POD_DEPLOYMENT" == true ];
    cp "$cfgmap_path/templates/config-data-template.yaml" $new_gen_file
else
    cp "$cfgmap_path/templates/config-template.yaml" $new_gen_file
fi
```

Reply:
fixed with deployment mode approach